applications/OptimizationApplication/python_scripts/algorithms/algorithm_steepest_descent.py

```diff
@@ -101,22 +102,23 @@ def Solve(self):
                         algorithm_data.GetBufferedData()["abs_obj[%]"] = self.__objective.GetAbsoluteChange() / initial_value * 100
                     print(self.__objective.GetInfo())
 
-                with TimeLogger("Calculate gradient", None, "Finished"):
+                with TimeLogger("AlgorithmSteepestDescent::Calculate gradient", None, "Finished"):
                     obj_grad = self.__objective.CalculateStandardizedGradient()
 
-                with TimeLogger("Calculate design update", None, "Finished"):
+                with TimeLogger("AlgorithmSteepestDescent::Calculate design update", None, "Finished"):
                     search_direction = self.ComputeSearchDirection(obj_grad)
                     algorithm_data.GetBufferedData()["search_direction"] = search_direction
-
                     alpha = self.__line_search_method.ComputeStep()
-
                     update = search_direction * alpha
                     self.__control_field += update
+                    print(self.__line_search_method.GetInfo())
```

Comment:
```suggestion
                    Kratos.Logger.PrintInfo("AlgorithmSteepestDescent", "\n" + self.__line_search_method.GetInfo())
```
                    Kratos.Logger.PrintInfo("AlgorithmSteepestDescent", "\n" + self.__objective.GetInfo())